data/helpers/usersHelpers.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../dbconfig");
+
+module.exports = {
+  getUserByName,
+  addUser,
+  getAllUsers
+};
+
+function getUserByName(username) {
+  console.log(username);
+  return db("users")
+    .where(username) //{ username: 'user2' }
```

Comment:
This function (with its name) might be better suited for taking the string (the name) directly and just using that to pass in an object, like so:
```javascript
console.log(username); // 'user2'
return db("users")
  .where({ username })
```